app/src/main/res/layout/activity_details_movie.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<ScrollView
+    android:fillViewport="true"
+    android:id="@+id/movie_view"
+    android:layout_height="match_parent"
+    android:layout_width="match_parent"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <LinearLayout
+        android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:orientation="vertical"
+        android:background="@android:color/white">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:gravity="center_horizontal">
+
+            <ImageView
+                android:id="@+id/movie_back_drop_path"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:scaleType="centerInside"
+                android:adjustViewBounds="true"
+                android:layout_gravity="center"/>
+
+        </LinearLayout>
+
+        <!-- Details layout -->
+        <LinearLayout
+            android:background="@android:color/white"
+            android:id="@+id/movie_details_layout"
+            android:layout_height="wrap_content"
+            android:layout_width="match_parent"
+            android:orientation="horizontal"
+            android:paddingBottom="20dp"
+            android:paddingLeft="21dp"
+            android:paddingRight="30dp">
+
+            <ImageView
+                android:id="@+id/movie_cover"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="25dp"
+                android:layout_width="wrap_content" />
+
+            <LinearLayout
```

Comment:
Do you see any issue with having nested `ViewGroup`s? What would be your strategy to avoid that?

Reply:
Yes, I find it problematic to use nested viewGroups, this increases the complexity of the xml, in this case I chose to do this for convenience even in relation to time, since I worked longer with LinearLayout for example, but the ideal solution in this case would be ConstraintLayout